salt/modules/freebsd_sysctl.py

```diff
@@ -176,6 +176,6 @@ def persist(name, value, config="/etc/sysctl.conf"):
     with salt.utils.files.fopen(config, "w+") as ofile:
         nlines = [salt.utils.stringutils.to_str(_l) + "\n" for _l in nlines]
         ofile.writelines(nlines)
-    if config != "/boot/loader.conf":
+    if config != "/boot/loader.conf.local":
```

Comment:
obviously this should match whatever value is in `_formatfor`. It might be worthwhile setting some "constants", e.g. `LOADER_CONF = '/boot/loader.conf'` or something.